cfme/tests/ansible/test_embedded_ansible_services.py

```diff
@@ -130,6 +129,12 @@ def test_service_ansible_playbook_crud(ansible_repository):
     assert not cat_item.exists
 
 
+def test_service_ansible_playbook_tagvis(request, ansible_catalog_item):
```

Comment:
Fulfill tests steps, as for me it is not clear if tag visibility is tested or just tagging